Composer/packages/tools/language-servers/language-generation/resources/memoryVariables.json

```diff
@@ -0,0 +1,13 @@
+{
+    "this" : [{"value": ""}, {"turnCount": ""}],
```

Comment:
this structure is a little wired, why not

"this": {
  "value": ""
  "turnCount":""
}

Reply:
Also, can you work with @luhan2017 to make sure this list is as rich as possible.